stl/inc/thread

```diff
@@ -196,10 +196,14 @@ namespace this_thread {
             if (_Abs_time <= _Now) {
                 return;
             }
-
-            _timespec64 _Tgt;
-            (void) _To_timespec64_sys_10_day_clamped(_Tgt, _Abs_time - _Now);
-            _Thrd_sleep(&_Tgt);
+            constexpr chrono::milliseconds _Clamp{chrono::hours{24}};

```

Comment:
I think there is no 10-day limit to `Sleep`. As there have to be a clamp to avoid integer overflow etc, I've choosed one day as clamp value. This is a bit arbitary, but I think it should work well.